lib/vue-lib/src/design-system/icons/Icon.vue

```diff
@@ -115,6 +118,17 @@ const AUTO_SPIN_ICONS_COUNTER_CLOCKWISE = ["refresh-active"];
       @apply animate-spin-cc;
     }
   }
+  &.--spin-slow-with-horizontal-flip {
+    transform: scaleY(-1);
+
+    > :deep(svg) {
+      // Note that we spin counter-clockwise because we've flipped it. That's also why we don't
+      // have the flip and the spin separated: it would be confusing to the caller to request
+      // counter-clockwise and flip when they truly want clockwise.
+      @apply animate-spin-cc;
+      animation-duration: 5s;
+    }
+  }
```

Comment:
Why are these grouped and not just two options? If you flip, you must also invert spin direction... and we want it slower for this one too!

@VictoriaMolgado this uses your `5s` duration, by the way.